FXSkins/src/main/resources/com/pixelduke/control/skin/fx-skins.css

```diff
@@ -47,23 +47,31 @@ TextField {
     -fx-skin: "com.pixelduke.control.skin.FXTextFieldSkin";
 }
 
-TextField {
-    -right-button-visible: true;
+.text-field {
+    -fx-padding: 0.3333333em 3em 0.3333333em 0.5833333em;
 }
 
-TextField > .right-button > .right-button-graphic {
+.text-field > .right-button > .right-button-graphic {
     -fx-shape: "M221.738,305.873l6.135,6.16l-2.875,2.863l-6.135-6.159l-6.263,6.237l-2.864-2.875l6.263-6.238l-6.177-6.202l2.875-2.863l6.177,6.201l6.244-6.22l2.864,2.876L221.738,305.873z";
 
     -fx-padding: 0.416667em;
-    -fx-background-color: #333;
+    -fx-background-color: #111;
 }
 
-TextField > .right-button:hover {
+.text-field > .right-button {
     -fx-cursor: default;
+
+    -fx-background-color: transparent;
+    -fx-background-insets: -0.3333333em -0.6666666em -0.3333333em -0.5833333em; /* 4 8 4 7 px in em */
```

Comment:
Those are negative insets, so they actually expand the button. Otherwise it would look like this:
![insets](https://user-images.githubusercontent.com/18708736/115231402-cd8eb900-a115-11eb-868e-9f8b429c9cdc.png)

Reply:
The problem is that the stackpane does not grow

Reply:
I think that, probably, if we change the layoutchidlren method so that the button covers the textfield height we won't need to be changing the button insets to make it appear bigger.. 
As long as we position and resize the button to have the appropriate dimensions I think there won't be any need for any additional tweaks to make it bigger.

Reply:
I checked again and you are right on this. It can be removed if we position it right